utils/weight-diff/src/main.rs

```diff
@@ -270,10 +275,17 @@ fn main() {
     match command {
         Commands::Dump { output_path, label } => {
             let writer = fs::File::create(output_path).unwrap();
+            let vara_schedule: Schedule<vara_runtime::Runtime> = Default::default();
+            let process_costs = vara_schedule.process_costs();
```

Comment:
I wouldn't merge `Schedule` and `ProcessCosts` into one struct. We could dump `ProcessCosts` from the vara-runtime and add `serde` attributes to it. We might also have to add `serde` attributes to all `ProcessCosts` fields...

In this way we could implement the `process_costs` method, but in our lightweight scheduler (`utils/wasm-instrument/src/gas_metering/schedule.rs`).